settings/api/controller/update/controller.go

```diff
@@ -0,0 +1,33 @@
+package update
+
+import (
+	"fmt"
+	"log"
+	"encoding/json"
+	"github.com/julienschmidt/httprouter"
+	"net/http"
+	"api/model"
+	"github.com/GDGVIT/socrates/schema"
+)
+
+type Controller struct {
+	configModel *model.Model
+}
+
+func (ctrl *Controller) Update(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	config := &schema.Config{}
+	fmt.Printf("In update\n%+v\n", r.Body)
```

Comment:
Use a logging library (like logrus) to log stuff out, and in JSON format.

Reply:
Will do